hive-data-gen/src/main/java/com/wipro/ats/bdre/datagen/util/RandomValueGenerator.java

```diff
@@ -25,45 +25,44 @@
  *
  */
 public class RandomValueGenerator {
-	
 
-	public static Random random = new Random();
-	private static final Logger LOGGER=Logger.getLogger(RandomValueGenerator.class);
-
-	
+	public static final Random RANDOM = new Random();
+	public static final Logger LOGGER = Logger.getLogger(RandomValueGenerator.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make the "LOGGER" logger private static final. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1312)